### ISSUE TITLE
Pie Charts - Override Label Threshold

### ISSUE THREAD

**Issue by lucabrasi83** [swimlane/ngx-charts#880]
**I'm submitting a ...**  (check one with "x")

- [ ] bug report - search github for a similar issue or PR before submitting
- [ X] feature request
- [ ] support request - use StackOverflow (add the `ngx-charts` tag) or the gitter chat for support questions

**Current behavior**
I understand as per design, labels are not shown below a certain threshold to avoid overlapping.

However, in some instances we could override this behaviour when only two values are shown in the chart.

Example below where we just show Online vs Offline devices:

![screen shot 2018-06-18 at 14 05 59](https://user-images.githubusercontent.com/21274414/41522631-08396d84-7301-11e8-9c3e-0fd97d77fc94.png)

**What is the motivation / use case for changing the behavior?**

Ability to show labels on minimum amount of values in a pie chart

**Please tell us about your environment:**
WebStorm, NPM

* **ngx-charts version:** x.x.x
8.0.1

* **Angular version:** x.x.x
6.0.2

* **Browser:** [all | Chrome XX | Firefox XX | IE XX | Safari XX | Mobile Chrome XX | Android X.X Web Browser | iOS XX Safari | iOS XX UIWebView | iOS XX WKWebView ]

All

* **Language:** [all | TypeScript X.X | ES6/7 | ES5]

Typescript 2.7.2

**Comment by marjan-georgiev** [comment]
Makes sense. I would add an input `showLabelThreshold` that accepts a number in the range [0, 100] indicating percentage of total value, above which the labels would be shown.

**Comment by lucabrasi83** [comment]
Thanks @marjan-georgiev . I think it's fair to have a number in a range, so we can adjust the threshold based on the parent container size and number of elements to render in the chart.

**Comment by frankdavidcorona** [comment]
@marjan-georgiev Any plannig to release this PR. Thanks.

**Comment by originals** [comment]
would love this feature

**Comment by AlexAvatar88** [comment]
Need this feature too!

**Comment by AbdaalZulfi** [comment]
any update on this PR????
